test/export/test_torchbind.py

```diff
@@ -111,7 +111,59 @@ def __init__(self):
                 self.attr = torch.classes._TorchScriptTesting._Foo(10, 20)
 
             def forward(self, x):
-                return x + torch.ops._TorchScriptTesting.takes_foo(self.attr, x)
+                a = torch.ops._TorchScriptTesting.takes_foo(self.attr, x)
+                b = torch.ops._TorchScriptTesting.takes_foo(self.attr, a)
+                return x + b
+
+        m = MyModule()
+        input = torch.ones(2, 3)
+        with enable_torchbind_tracing():
+            ep = torch.export.export(m, (input,), strict=False)
+
+        unlifted = ep.module()
+        self.assertEqual(m(input), unlifted(input))
+
+        with enable_torchbind_tracing():
```

Comment:
Are we always expected to call this context manager whenever there is TorchBind object in the graph?

Reply:
For now, with suo's hacks yes. But I think Yidi's working on a legit thing